src/Message/PurchaseRequest.php

```diff
@@ -41,11 +41,57 @@ public function setEntranceCode($value)
         return $this->setParameter('entranceCode', $value);
     }
 
+    public function getMakeInvoice()
+    {
+        return $this->getParameter('makeInvoice') ?: 'true';
+    }
+
+    public function setMakeInvoice($value)
+    {
+        return $this->setParameter('makeInvoice', $value);
+    }
+
+    public function getMailInvoice()
+    {
+        return $this->getParameter('mailInvoice') ?: 'true';
```

Comment:
In our experience it's what most of our customers want, although the default according to sisow is that it is off, so I can see your point.